test-it/ViaTests/DateFormatString.via

```diff
@@ -44,6 +44,10 @@ define(HelloTime dv(.VirtualInstrument (
 	Printf("11:(%s)\n" s1)
 	FormatDateTimeString(s1 "%Z"  t4 true)
 	Printf("12:(%s)\n" s1)
+	FormatDateTimeString(s1 * t1 true)
+	Printf("13:(%s)\n" s1)
+	FormatDateTimeString(s1 * * true) // Output is going to be different everytime, so we don't print the result
+	FormatDateTimeString(s1 * * *) // Same here, we want to exercise the function with 'default' values
```

Comment:
Instead of not printing, yYou can make the output print with "//" at the beginning of the line and the test drivers will ignore the difference, but you can still see the output if you run the test manually.
Some of the tests do this to print elapsed time for tests, which varies per run but doesn't cause failure.